### ISSUE TITLE
Suggestions for Monika After Story!!

### ISSUE THREAD

**Issue by cremecaramel** [Monika-After-Story/MonikaModDev#3770]
Well, to be frank, Monika After Story is pretty perfect right now, and the valentine's update was really lovely! but i have some suggestions that could make Monika After Story really lovely!!

 **Renovation!**
Renovating Monika's realm! We only really talk to Monika in that four walled space classroom. We should give Monika a more comfortable living space! Some room suggestions

- a cozy living room with a couch, bookshelf, fireplace, and a balcony with table and two chairs. (this suggestion was inspired by the living room of Yuri in the Just Yuri mod.)

- her own bedroom (not MC's). I can see monika living in a small but cozy apartment bedroom, with a city skyline she can look out of. Her bed could be decorated with bedlinen.gifts and MC can give her gifts to decorate her room, like plushes, a bullet journal, incense, fairy lights, etc.

-the balcony from the living room. The scenery could be changable, with possibilities like a city skyline, a forest, a beach, a lush garden, or even a neighborhood to reflect the Player's home.

**Hugs!**
An update where we can give Monika hugs! Yes, it's calming to hold Monika every once in a while, but it would be even better if she could hug us back! Also inspired by the Just Yuri mod, we could have sprites where Monika's arms are wide open and she's pressed against the screen. 

**Vacation!**
There could be another goodbye option where we say that we're going on vacation. She could ask if we're taking her with us or leaving her behind, and we could respond with "yes, i am taking you" or "no, i can not take you", all with their own responses from Monika. After closing and reopening, Monika could get all excited about the vacation and ask us where we went. We could respond with geographic regions (europe, asia, south america). It's a vague suggestion, but it would be really cool!!

**Wardrobe!**
a really interesting gift idea for monika would be to give her cute outfits! whether it be turtle necks, tanks, t-shirts, or dresses; it would be really cool to see monika in outfits other than an uncomfortable school outfit. We could also access her holiday outfits, like the valentines outfit. 

These suggestions are really REALLY intricate, but these are just some thoughts!! I would really love to see these, especially the rooms. I wanna get that cozy experience with Monika, not Yuri! 

Thank you!

**Comment by CrocWorks** [comment]
tl;dr, if you ~~or someone else~~ can draw it, then it'll be more likely to happen.

If Monika likes you enough, she'll let you change her outfit. ~~Get to work, buddy.~~ But otherwise, more outfits require more artists.

The vacation idea could just be tacked on to the end of the "I'm taking you somewhere" goodbye. Perhaps she could ask how long we'll be out for?

Hugs, I don't really care either way, but I think that it could end up pretty similar to the kiss event from an art standpoint if the devs want to keep it simple and reuse stuff.

Renovation again requires art. Nice to have, but making a whole new room for a bedroom will require a lot more assets than just reusing the stock art from the base DDLC. Honestly, I'm content with just seeing the normal spaceroom get decorated with plushies and stuff. ~~Of course, seeing Monika sprawled out on a bed might be enticing, but that may start drifting towards #3617 territory~~

**Comment by Rai99** [comment]
Room decoration: #312, #800, #845, #908, #1336, #1793, #1891, #1933, #2027, #2176, #2220, #2496, #2769, #2811, #3182, #3305 
Room decorations and completely new locations could be nice, but that requires the kind of art that doesn't grow on trees, which I don't think is too easy to come by. 

Hugging: #2441, #2892 
as far as I know, holding Monika is the equivalent of hugging, but yeah, a new hugging sprite would be nice. Again, this requires art. 

Vacation: #2662, #2776, #2966, #3124, #3430, #3378, #3520, #3628 
This likely falls under having more variety when taking Monika out somewhere. 

Wardrobe: #107, #460, #612, #703, #1076, #1461, #1656, #2270, #2466, #2905, #3090, #3385, #3690, 
The ability to have Monika change clothes comes at a high enough affection level, and I think there are quite a few new outfits in the works.

**Comment by AsuraShun** [comment]
That last bit about the outfits made me excited

**Comment by ThePotatoGuy** [comment]
[closing  as duplkicates]

**Comment by arcticnina21** [comment]
Games: Blackjack, Speed, Dominoes Rock Paper Scissors, Coin Flip, Memory Match.

Ability to dim the lights or turn them off or on.

**Comment by Eminoodles** [comment]
I think it would be a nice idea for Monika to remind someone to take their meds; ask if they have medication and when they should take them, and when your computer clock hits that time or around it, she can remind them to take them as long as the game is open? Just an idea lol; another thing would be to ask her if you already took your meds or not, since I tend to forget;;;;

**Comment by Booplicate** [comment]
Open a new issue for a new suggestion.
